### PR TITLE
SSD ONNX model failed on CPU after switching to NMS-5 in onnx_importer

### PR DESCRIPTION
**Pull request by vgavrilo**
SSD ONNX model failed on CPU after switching to NMS-5 in onnx_importer

Ticket: #42779

### CODE REVIEW

**Comment by AndrewBakalinIntel** [comment]
As this model is currently disabled in testing, could you please also revert the changes that were made in onnx python tests to enable this model again? These are changes: https://github.com/openvinotoolkit/openvino/pull/3003/commits/341c3ed2d447f9153aa12b0531d8f47b6ecc39ed

**Comment by vgavrilo** [comment]
> As this model is currently disabled in testing, could you please also revert the changes that were made in onnx python tests to enable this model again? These are changes: [341c3ed](https://github.com/openvinotoolkit/openvino/commit/341c3ed2d447f9153aa12b0531d8f47b6ecc39ed)

Done.

File: inference-engine/src/legacy_api/src/transformations/convert_opset1_to_legacy/convert_opset1_to_legacy.cpp [hunk 1]
```diff
@@ -139,6 +139,8 @@ bool ngraph::pass::ConvertOpSet1ToLegacy::run_on_function(std::shared_ptr<ngraph
     anchor->add_matcher<ngraph::pass::ConvertLSTMSequenceMatcher>();
     anchor->set_name("ngraph::pass::LegacyConversions");
 
+    manager.register_pass<ngraph::pass::ConstantFolding>();
```
Comment (GlebKazantaev) [inline inference-engine/src/legacy_api/src/transformations/convert_opset1_to_legacy/convert_opset1_to_legacy.cpp:142]: CF is a very time consuming operation. If we know which pass requires CF let's do it manually. You can find an example here: https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/transformations/src/transformations/common_optimizations/lin_op_sequence_fusion.cpp#L17

**Review by GlebKazantaev — CHANGES_REQUESTED** [review]

File: inference-engine/src/legacy_api/src/transformations/convert_opset1_to_legacy/convert_opset1_to_legacy.cpp (hunk 1 above)
Comment (vgavrilo) [reply inference-engine/src/legacy_api/src/transformations/convert_opset1_to_legacy/convert_opset1_to_legacy.cpp]: Started to write the transformation to fold input "K" for `TopK`.

**Comment by lazarevevgeny** [comment]
Not applicable anymore

**Pull request closed without merging**